internal/report/report.go

```diff
@@ -433,7 +433,19 @@ func PrintAssembly(w io.Writer, rpt *Report, obj plugin.ObjTool, maxFuncs int) e
 	}
 
 	if len(syms) == 0 {
-		return fmt.Errorf("no matches found for regexp: %s", o.Symbol)
+		if address == nil { // User provides a symbol instead of an address
+			if len(symbols) == 0 {
+				return fmt.Errorf("no matches found for regexp %s in binaries", o.Symbol)
```

Comment:
I would use singular "in the binary", I think plural can be confusing to the user. When the user specifies the symbol or address they think of a single binary that should contain that pattern, not multiple binaries.

Reply:
Done.